frontend/src/components/SignUp.js

```diff
@@ -192,7 +192,11 @@ export class SignUp extends React.Component {
                         </div>
                         <ReCAPTCHA
                             ref={this.recaptchaRef}
-                            sitekey={process.env.REACT_APP_CODALAB_RECAPTCHA_SITE_KEY}
+                            sitekey={
+                                process.env.NODE_ENV === 'production'
```

Comment:
1. Is it possible to just always use `window.env.REACT_APP_CODALAB_RECAPTCHA_SITE_KEY`, even for development? This would simplify the code a bunch. We could do this by modifying the command in docker-compose.dev.yml to include `node setup-frontend.js` in it.
2. If we can do 1., then we can actually remove the REACT_APP_ prefix altogether (so we can do something like `window.env.CODALAB_RECAPTCHA_SITE_KEY`, further simplifying env variable logic).

Reply:
Actually, let's not do suggestion 2 (removing the REACT_APP_ prefix). This is because then, in setup-frontend.js, we'd have no way to distinguish between frontend (non-secret) and non-frontend (secret) env variables

Reply:
Just applied suggestion 1.